### PR TITLE
gtk+3: Build a bottle for Linuxbrew

### PR DESCRIPTION
**Pull request by iMichka**
This is an automated pull request to build a new bottle for linuxbrew-core
based on the existing bottle block from homebrew-core.

### CODE REVIEW

**Comment by iMichka** [comment]
```
==> make install
...
Last 150 lines from /root/.cache/Homebrew/Logs/gtk+3/02.make:
/home/linuxbrew/.linuxbrew/Cellar/at-spi2-atk/2.32.0/lib/libatk-bridge-2.0.so: undefined reference to `dbus_message_get_interface@LIBDBUS_1_3'
/home/linuxbrew/.linuxbrew/Cellar/at-spi2-atk/2.32.0/lib/libatk-bridge-2.0.so: undefined reference to `dbus_bus_add_match@LIBDBUS_1_3'
/home/linuxbrew/.linuxbrew/Cellar/at-spi2-atk/2.32.0/lib/libatk-bridge-2.0.so: undefined reference to `dbus_server_listen@LIBDBUS_1_3'
/home/linuxbrew/.linuxbrew/Cellar/at-spi2-atk/2.32.0/lib/libatk-bridge-2.0.so: undefined reference to `dbus_message_new_method_call@LIBDBUS_1_3'
```

**Comment by iMichka** [comment]
```
Did not find CMake 'cmake'
Found CMake: NO
Run-time dependency xkbcommon found: NO (tried pkgconfig)
```

**Comment by iMichka** [comment]
```
Run-time dependency wayland-protocols found: NO (tried pkgconfig and cmake)

meson.build:539:2: ERROR: Dependency "wayland-protocols" not found, tried pkgconfig and cmake
```

**Comment by iMichka** [comment]
```
/home/linuxbrew/.linuxbrew/Cellar/at-spi2-atk/2.32.0/lib/libatk-bridge-2.0.so: undefined reference to `dbus_connection_close@LIBDBUS_1_3'
/home/linuxbrew/.linuxbrew/Cellar/at-spi2-atk/2.32.0/lib/libatk-bridge-2.0.so: undefined reference to `dbus_message_get_args@LIBDBUS_1_3'
/home/linuxbrew/.linuxbrew/Cellar/at-spi2-atk/2.32.0/lib/libatk-bridge-2.0.so: undefined reference to `dbus_message_iter_append_basic@LIBDBUS_1_3'
```

**Comment by iMichka** [comment]
```
ls /home/linuxbrew/.linuxbrew/Cellar/at-spi2-atk/2.32.0/lib/
gnome-settings-daemon-3.0  gtk-2.0  libatk-bridge-2.0.so  libatk-bridge-2.0.so.0  libatk-bridge-2.0.so.0.0.0  pkgconfig

ls /home/linuxbrew/.linuxbrew/Cellar/dbus/1.12.16/lib
cmake  dbus-1.0  libdbus-1.a  libdbus-1.so  libdbus-1.so.3  libdbus-1.so.3.19.11  pkgconfig

brew linkage at-spi2-atk
System libraries:
  /lib/x86_64-linux-gnu/libc.so.6
  /lib/x86_64-linux-gnu/libpthread.so.0
Dependencies with no linkage:
  at-spi2-core
  atk

brew linkage at-spi2-core
System libraries:
  /lib/x86_64-linux-gnu/libc.so.6
Homebrew libraries:
  /home/linuxbrew/.linuxbrew/lib/libatspi.so.0 (at-spi2-core)
  /home/linuxbrew/.linuxbrew/lib/libdbus-1.so.3 (dbus)
  /home/linuxbrew/.linuxbrew/lib/libgio-2.0.so.0 (glib)
  /home/linuxbrew/.linuxbrew/lib/libglib-2.0.so.0 (glib)
  /home/linuxbrew/.linuxbrew/lib/libgobject-2.0.so.0 (glib)
  /home/linuxbrew/.linuxbrew/lib/libX11.so.6 (linuxbrew/xorg/libx11)
  /home/linuxbrew/.linuxbrew/lib/libXtst.so.6 (linuxbrew/xorg/libxtst)
Dependencies with no linkage:
  linuxbrew/xorg/xorgproto

brew linkage atk      
System libraries:
  /lib/x86_64-linux-gnu/libc.so.6
Homebrew libraries:
  /home/linuxbrew/.linuxbrew/lib/libglib-2.0.so.0 (glib)
  /home/linuxbrew/.linuxbrew/lib/libgobject-2.0.so.0 (glib)

nm /home/linuxbrew/.linuxbrew/Cellar/at-spi2-atk/2.32.0/lib/libatk-bridge-2.0.so | grep dbus_connection_close            
                 U dbus_connection_close@@LIBDBUS_1_3
```

**Comment by iMichka** [comment]
This will need a rebuild of at-spi2-core and at-spi2-atk #14310

File: Formula/gtk+3.rb [hunk 1]
```diff
@@ -24,8 +24,13 @@ class Gtkx3 < Formula
   depends_on "libepoxy"
   depends_on "pango"
   unless OS.mac?
+    depends_on "libxslt" => :build # for xsltproc
+    depends_on "cmake"
```
Comment (maxim-belkin) [inline Formula/gtk+3.rb]: `depends_on "cmake" => :build`

File: Formula/gtk+3.rb [hunk 2]
```diff
@@ -24,8 +24,13 @@ class Gtkx3 < Formula
...
     depends_on "at-spi2-atk"
     depends_on "cairo"
+    depends_on "iso-codes"
+    depends_on "libxkbcommon"
+    depends_on "linuxbrew/xorg/wayland-protocols"
```
Comment (maxim-belkin) [inline Formula/gtk+3.rb]: I _think_ this should be a build-time dependency

**Pull request closed without merging**